_drafts/oryx/packages.md

```diff
@@ -0,0 +1,99 @@
+---
+title: Oryx packages
+description: Use Oryx packages from npm to ensure you can easily upgrade to newer versions.
+last_updated: Apr 19, 2023
+template: concept-topic-template
+---
+
+While the Oryx code base is [available on Github](https://github.com/spryker/oryx/), the code is published and distributed as so-called packages on npm. npmjs.com is a widely used registry that is used to packages. Package managers (e.g. npm, yarn, deno, bun, etc) can be used to install the dependencies in a project. The dependencies are typically configured in the package.json file of your application.
+
+The Oryx [packages](./packages.md) are distributed on under the [spryker-oryx](https://www.npmjs.com/org/spryker-oryx) organization. Each time a new version is published, the version number is bumped, see versioning for more information on the version strategy
+
+<!-- Add link to version.md (see https://spryker.atlassian.net/browse/HRZ-2147) -->
+
+You are encourage to [install](./set-up-oryx.md) the packages to your application rather than cloning the Oryx repository. By depending on packages, you can upgrade to later versions of the packages going forward with little effort.
+
+## Layers
+
+While packages are all distributed as a flat list, there is an architectural hierarchy. The hierarchy is important as it will protect from cyclic dependencies. Packages inside a layer can depend on sibling packages inside the layer without any issues. Packages can never depend on a layer above.
+
+While the package layering might not be relevant during your development, it might help you to better understand the package dependencies. The diagram below shows you the four package layers, the top layer in the diagram is the [boilerplate application](./boilerplate.md) that uses a preset to setup the a standard application.
+
+<!-- Add link to presets.md (see https://spryker.atlassian.net/browse/HRZ-2153) -->
+
+```mermaid
+%%{init: {'theme':'base', 'themeVariables': { 'primaryColor': '#fff','primaryBorderColor': '#aaa'}}}%%
+
+flowchart TD
+classDef app fill:#fff,stroke:#fff;
+
+app("Application"):::app
+template(Template packages):::template
+domain(Domain packages)
+platform(Platform packages)
+base(Base packages)
+
+app-..->|"use presets\n(e.g. b2c)"|template
+app-->domain
+
+template --> domain
+domain --> platform
+platform --> base
+domain --> base
+```
+
+### Template packages
+
+The template layer contains packages that can be used as quick starters for demos and projects. Templated packages do follow semantic versioning and will deliver on the promise of upgradability, but are fairly opinionated as they mainly serve to get up to speed fast with a standard frontend application.
+
+| Packages                                                       | Location                    |
```

Comment:
We are already having `cli` and `create-oryx`.

Reply:
I deliberately left them out as they're pretty fresh.